diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -98,19 +100,90 @@ impl ToSql<Timestamp, Sqlite> for NaiveDateTime {
     }
 }
 
+impl FromSql<TimestampWithTz, Sqlite> for NaiveDateTime {
+    fn from_sql(value: backend::RawValue<'_, Sqlite>) -> deserialize::Result<Self> {
```

Comment:
This is pretty much copy pasta of the `impl FromSql<Timestamp, Sqlite> for NaiveDateTime` with fewer allowable formats (only the ones ending in `Z` or `+/-HOURS`. Happy tor refactor this code to a common util method or leave it alone for now.